node/src/service.rs

```diff
@@ -61,6 +61,7 @@ type HostFunctions = (
 ///
 /// Use this macro if you don't actually need the full service, but just the builder in order to
 /// be able to perform chain operations.
+#[allow(clippy::type_complexity)]
```

Comment:
This line falls into one of the files that we need to keep updated with the upstream. We have a constant burden upgrading our code to be compatible with the upstream. So in general I'm in favour of making minimal changes to such places to make the upgrade easier.

Reply:
@aliXsed we could try to get in included upstream, where does this file come from? I did not find it on master in substrate